lib/routes/me.js

```diff
@@ -89,7 +89,8 @@ module.exports = function(crowi, app) {
         return res.render('me/index', {});
       }
 
-      User.findOne({email: email}, (err, existingUserData) => {
+      User.findOne({email: email, _id: {'$ne': userData._id}}, (err, existingUserData) => {
```

Comment:
hmm ... ?
if the user couldn't find here, then the process is not into this if statement, 
it means ... lang and name are also not updated?

Reply:
it should be ...

- user changes their email address to another one, and it's not duplicated => email can be changed
- user changes email address and it's duplicated => email can't be updated
- anyway, others (name and lang) can be updated

Reply:
First of all, we should write exception handling, right?😉

Reply:
when email was invalid, `lang` and `name` are not updated.
https://github.com/crowi/crowi/pull/259/files#diff-1972c53f45200d68ac6d5e645409aca1L87

here is same logic.

Reply:
I misunderstood about this logic.
It would be ok.